### PR TITLE
添加图片模型

### PR DESCRIPTION
**Pull request by JinhuW**
- 添加Select 按钮来实现模型的选择
- 添加图片生成模型

### CODE REVIEW

**Comment by JinhuW** [comment]
@Chanzhaoyu 
不好意思 commit message 有点乱 
我本地尝试rebase 了 但是一直是乱码 所以放弃了

**Comment by oasis168** [comment]
很好的功能，别放弃，再研究研究啊

**Comment by JinhuW** [comment]
> @Chanzhaoyu 不好意思 commit message 有点乱 我本地尝试rebase 了 但是一直是乱码 所以放弃了

我说的 放弃 只是放弃 清理 commit message 了

**Comment by JinhuW** [comment]
image

**Comment by oasis168** [comment]
> > @Chanzhaoyu 不好意思 commit message 有点乱 我本地尝试rebase 了 但是一直是乱码 所以放弃了
> 
> 我说的 放弃 只是放弃 清理 commit message 了

那就好，吓我一跳，哈哈

**Comment by renwofei423** [comment]
> > @Chanzhaoyu 不好意思 commit message 有点乱 我本地尝试rebase 了 但是一直是乱码 所以放弃了
> 
> 我说的 放弃 只是放弃 清理 commit message 了

gitlab合并的时候可以整理commit message，不清楚 github行不行

**Review by 53jk1 — APPROVED** [review]

**Comment by zaiMoe** [comment]
> 

你可以先 rebase 成一个 commit， 然后在 rebase 一下 master。 另外可以补充一下使用说明吗？感觉很不错的功能

**Comment by zaiMoe** [comment]
> > @Chanzhaoyu 不好意思 commit message 有点乱 我本地尝试rebase 了 但是一直是乱码 所以放弃了
> 
> 我说的 放弃 只是放弃 清理 commit message 了

你可以先将你这个分支的 commit 通过 rebase 合并成一条， 然后在 rebase 一下 master， 处理冲突就行啦。 另外可以补充一下使用说明吗？感觉很不错的功能。

File: service/src/index.ts [hunk 1]
```diff
@@ -19,21 +20,45 @@ app.all('*', (_, res, next) => {
...
-        res.write(firstChunk ? JSON.stringify(chat) : `\n${JSON.stringify(chat)}`)
-        firstChunk = false
-      },
-      systemMessage,
-    })
+
+    if (options.apiModel === 'stable-diffusion') {
+      const prediction = await replicate
+        .model(
+          'stability-ai/stable-diffusion:db21e45d3f7023abc2a46ee38a23973f6dce16bb082a930b0c49861f96d1e5bf',
```
Comment (zaiMoe) [inline service/src/index.ts:34]: 这一串是不是使用者配置？

Comment (zaiMoe) [inline service/src/index.ts:34]: 这哥是不是让使用者自己配置？

**Comment by shuncon** [comment]
请问一下这个在chatgpt-web2.10.7版本怎么使用，是以英文形式提交，让后在绘画出来嘛
但是我得到的结果是：
I'm sorry, but as an AI language model, I am not capable of drawing pictures. However, I can provide some general tips on how to draw an anime-style cat:

Start with a rough sketch of the cat's head, including the ears, eyes, nose, and mouth. Anime-style cats typically have larger eyes than real-life cats, so you may want to emphasize this feature.

Draw the cat's body, paying attention to the proportions and overall shape. Anime-style cats are often depicted with slender, elongated bodies and tails.

Add details such as fur texture, markings, and accessories like collars or bows.

Color the cat using bright, vibrant hues commonly seen in anime art styles.

Remember, there is no one "correct" way to draw an anime-style cat. Experiment with different styles and techniques to find what works best for you.

**Comment by JinhuW** [comment]
> 请问一下这个在chatgpt-web2.10.7版本怎么使用，是以英文形式提交，让后在绘画出来嘛 但是我得到的结果是： I'm sorry, but as an AI language model, I am not capable of drawing pictures. However, I can provide some general tips on how to draw an anime-style cat:
> 
> Start with a rough sketch of the cat's head, including the ears, eyes, nose, and mouth. Anime-style cats typically have larger eyes than real-life cats, so you may want to emphasize this feature.
> 
> Draw the cat's body, paying attention to the proportions and overall shape. Anime-style cats are often depicted with slender, elongated bodies and tails.
> 
> Add details such as fur texture, markings, and accessories like collars or bows.
> 
> Color the cat using bright, vibrant hues commonly seen in anime art styles.
> 
> Remember, there is no one "correct" way to draw an anime-style cat. Experiment with different styles and techniques to find what works best for you.


很大的可能是 你没有清理缓存 

里面有一个下拉菜单 选择模型

**Comment by JinhuW** [comment]
> > image
> 
> 
> 
> 如何让gpt生成图片，是我的版本不够新吗

你得切换一个模型

**Comment by rzarvirov** [comment]
Interesting! How can I try this?

**Comment by oasis168** [comment]
为什么关闭呢？

**Pull request closed without merging**